membership-attribute-service/app/services/GuardianPatronService.scala

```diff
@@ -58,9 +55,9 @@ class GuardianPatronService(
       } yield accountDetailsFromStripeSubscription(subscription, paymentDetails, stripePatronsPublicKey)
     }
 
-  private def billingPeriodFromInterval(interval: String): RecurringPeriod = interval match {
-    case "year" => Year
-    case _ => Month
+  private def billingPeriodFromInterval(interval: String): ZBillingPeriod = interval match {
+    case "year" => ZYear
+    case _ => ZMonth
```

Comment:
we used to convert the low level stripe API data into our high level model, however since the high level model is removed, we need to produce the low level model instead.
(This also means adding a fake Guardian Patron to the catalog, so that it can find everything on the fly.)